### PR TITLE
Update v17 Release notes with a Breaking Change

### PR DESCRIPTION
**Pull request by MrFabio**
## Description

Added a note to the Breaking Changes for the updated version of `init_db.sql`

## Related Issue(s)

https://github.com/vitessio/vitess/pull/12206

## Checklist

-   [x] "Backport to:" labels have been added if this change should be back-ported
-   [x] Tests were added or are not required
-   [x] Did the new or modified tests pass consistently locally and on the CI
-   [x] Documentation was added or is not required

## Deployment Notes

None.

### CODE REVIEW

**Review by mattlord — APPROVED** [review]
LGTM. Thanks, @MrFabio ! I've made the same change in the GitHub release page: https://github.com/vitessio/vitess/releases/tag/v17.0.0

**Review by frouioui — CHANGES_REQUESTED** [review]
Hello @MrFabio, thank you for this contribution. Can you also edit the `release_notes.md` file please?

Note to maintainers: we will need to update the release notes on the UI once this is merged.

**Comment by MrFabio** [comment]
> Hello @MrFabio, thank you for this contribution. Can you also edit the `release_notes.md` file please?
> 
> Note to maintainers: we will need to update the release notes on the UI once this is merged. EDIT: [#14215 (review)](https://github.com/vitessio/vitess/pull/14215#pullrequestreview-1664921469) @mattlord did it.

Done

**Review by frouioui — APPROVED** [review]
LGTM, thank you!

**Review by GuptaManan100 — APPROVED** [review]

**Pull request merged**
